keps/2724-topology-aware-scheduling/README.md

```diff
@@ -257,6 +264,137 @@ the "rack" topology domain (there is only one such Pod, so the requirement
 is satisfied trivially). The PodSet corresponding to the worker only
 prefers the "rack" topology domain, so it may fallback to "block".
 
+###### JobSet with ReplicatedJob instance topology
+
+To address [Story 2](#story-2) we are introducing PodSet Slices, which are
+subsets of the whole PodSet that are target to their own topology requirements.
+
+To keep solution generic (not only for JobSet) and allow to split PodSet into
+groups for other workload types, we decided to go with low-level API to allow
+user to directly specify PodSet Slice size.
+
+To achieve a result of each ReplicatedJob instance having its own topology
+domain user should align the slice size to "completions".
+
+**Example**:
+
+```yaml
+apiVersion: jobset.x-k8s.io/v1alpha2
+kind: JobSet
+metadata:
+  name: tas-example-jobset
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+spec:
+  replicatedJobs:
+  - name: leader
+    replicas: 1
+    template:
+      spec:
+        completions: 1
+        parallelism: 1
+        template:
+          metadata:
+            annotations:
+              kueue.x-k8s.io/podset-required-topology: cloud.provider.com/topology-block
+          spec:
+            containers:
+            - name: leader
+              image: registry.k8s.io/e2e-test-images/agnhost:2.53
+              args: ["pause"]
+  - name: workers
+    replicas: 2
+    template:
+      spec:
+        completions: 4
+        parallelism: 4
+        template:
+          metadata:
+            annotations:
+              kueue.x-k8s.io/podset-slice-required-topology: cloud.provider.com/topology-host
+              kueue.x-k8s.io/podset-slice-size: 4
+          spec:
+            containers:
+            - name: worker
+              image: registry.k8s.io/e2e-test-images/agnhost:2.53
+              args: ["pause"]
+```
+
+In this example there will be 8 (2 ReplicatedJob instances with 4 
+"completions" each) worker pods and we say that we want to split
```

Comment:
What if completions and parallelism are different values?
Do you want to say `2 ReplicatedJob instances with 4 "parallelism" each`?

Reply:
Yes, thanks, for spotting that!